Formula/socat.rb

```diff
@@ -1,8 +1,8 @@
 class Socat < Formula
   desc "SOcket CAT: netcat on steroids"
   homepage "http://www.dest-unreach.org/socat/"
-  url "http://www.dest-unreach.org/socat/download/socat-1.7.3.4.tar.gz"
-  sha256 "d9ed2075abed7b3ec9730ed729b4c8e287c502181c806d4487020418a6e2fc36"
+  url "http://www.dest-unreach.org/socat/download/socat-1.7.4.0.tar.gz"
+  sha256 "3f1e01a4af181c5d1b32b2986089282dc6a789d2aa4c3ac67070b27979195b72"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-only"
```

from `README`
```
license
-------

socat is distributed under the terms of the GNU GPLv2;
except for install-sh, which is copyright MIT, with its own license;

In addition, as a special exception, the copyright holder
gives permission to link the code of this program with
any version of the OpenSSL library which is distributed
under a license identical to that listed in the included
COPYING.OpenSSL file, and distribute linked combinations
including the two. You must obey the GNU General Public
License in all respects for all of the code used other
than OpenSSL. If you modify this file, you may extend this
exception to your version of the file, but you are not
obligated to do so. If you do not wish to do so, delete
this exception statement from your version.


    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, version 2 of the License

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program; if not, write to the Free Software
    Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
```

Reply:
cc @Rylan12

Reply:
We can't specify non-SPDX licenses or license exceptions.

We'll have to either go with `license "GPL-2.0-only"` or `license :cannot_represent`

Reply:
I see, then, let's go with `license "GPL-2.0-only"` for now (gonna check with spdx repo as well)

Reply:
spdx repo has many open issues related to `openssl exception` (which sounds like under their radar)